### PR TITLE
skip psp test consistently

### PR DESCRIPTION
**Pull request by p0lyn0mial**
This PR:
 - brings back disabled e2e test
 - changes Kube API server in a way that it stops serving `PodSecurityPolicies` resources.

Note:
We decided to disable PSP explicitly in the code instead of using `runtime-config` flag. At the moment the flag allows only for disabling the entire group, not individual resources. We don't want to disable the group as it contains critical resources like `PodDisruptionBudgets`.

### CODE REVIEW

**Comment by p0lyn0mial** [comment]
I was hoping that disabling `PSP` would reveal tests that were messing with `PSP`. Previously, the disabled `PSP` test was occasionally enabled which initiated that there are some other tests that must be adding `PSP` resources to the system. Surprisingly no tests are complaining when `PSP` is disabled.

**Comment by deads2k** [comment]
I would prefer to disable it via a flag so that our overall distro stays closer to upstream kube.

**Comment by p0lyn0mial** [comment]
> I would prefer to disable it via a flag so that our overall distro stays closer to upstream kube.

We decided to disable PSP explicitly in the code instead of using runtime-config flag. At the moment the flag allows only for disabling the entire group, not individual resources. We don't want to disable the group as it contains critical resources like PodDisruptionBudgets.

File: vendor/k8s.io/kubernetes/pkg/registry/policy/rest/storage_policy.go [hunk 1]
```diff
@@ -55,11 +55,13 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	storage["poddisruptionbudgets"] = poddisruptionbudgetStorage
 	storage["poddisruptionbudgets/status"] = poddisruptionbudgetStatusStorage
 
-	rest, err := pspstore.NewREST(restOptionsGetter)
```
Comment (deads2k) [inline vendor/k8s.io/kubernetes/pkg/registry/policy/rest/storage_policy.go]: In the quest for the simplest patch, is this actually just wrapping it in an `if false` block?

**Comment by p0lyn0mial** [comment]
closing this pull as we decided to actually change/improve the e2e test instead.

/close

**Pull request closed without merging**